### DISCUSSION TITLE
Is there any pre-requisite setup required for flutter?

### DISCUSSION THREAD

**Discussion by jagadish-pattanaik** [Abhith/piggy-flutter#237] [Q&A]
Is there any setup required such as a database or API etc for the flutter app?
Is the flutter app fully functional? or is there anything that needs to be done?
Is the flutter web fully functional?

**Comment by abhith** [comment]
> Is there any setup required such as a database or API etc for the flutter app?

Yes, the Database and the API requirements, the setup guides are here 👉 [Setup Guide](https://github.com/piggyvault/piggyvault/blob/main/docs/SETUP_GUIDE.md)

> Is the flutter app fully functional? or is there anything that needs to be done?

The flutter app is fully functional, we use it almost every day 🙂

> Is the flutter web fully functional?

The flutter web project is not started yet. The web project in the source code is **Angular**, and it came bundled with the [aspnetboilerplate](https://aspnetboilerplate.com/), which is used for the backend API project.
